File: cpp/2-5.cpp
#include "disp.h"//使用Display单元库
int main(){
if(iswin())msgbox("yes");else msgbox("no");//输出窗口状态信息
createwin();//建立窗口
if(iswin())msgbox("yes");else msgbox("no");//输出窗口状态信息
closewin();//关闭窗口
if(iswin())msgbox("yes");else msgbox("no");//输出窗口状态信息
createwin();//再次建立窗口
if(iswin())msgbox("yes");else msgbox("no");//输出窗口状态信息
closewin();//再次关闭窗口
if(iswin())msgbox("yes");else msgbox("no");//输出窗口状态信息
return 0;}

File: pas/2-5.pas
uses Display;//使用Display单元库
begin
if IsWin() then Msgbox('Yes') else Msgbox('No');//输出窗口状态信息
CreateWin();//建立窗口
if IsWin() then Msgbox('Yes') else Msgbox('No');//输出窗口状态信息
CloseWin();//关闭窗口
if IsWin() then Msgbox('Yes') else Msgbox('No');//输出窗口状态信息
CreateWin();//再次建立窗口
if IsWin() then Msgbox('Yes') else Msgbox('No');//输出窗口状态信息
CloseWin();//再次关闭窗口
if IsWin() then Msgbox('Yes') else Msgbox('No');//输出窗口状态信息
end.
